### PR TITLE
search again

### PR DESCRIPTION
**Pull request by Euro20179**
after the video ends if -sl is specified it will ask to make another search (-s alone does nothing)

### CODE REVIEW

**Comment by pystardust** [comment]
@Euro20179 
This is a great idea. What if we make this in built with option `-l`?
So we get a search loop and then if you hit escape it asks you for a search. If you hit escape again it exits.
Even if we implement it as you say, we can make it so that loop is made 1, so that giving option ` -s ` would suffice

File: ytfzf [hunk 1]
```diff
@@ -241,6 +243,8 @@ while getopts "LhDmdfxHarltU:" opt; do
 		r)
 			random_select=1
 			;;
+		s)	search_again=1
+			;;
```
Comment (pystardust) [inline ytfzf:247]: Add 
```
YTFZF_LOOP=1
```
So that it auto matically assumes loop when `-s` is given

**Comment by pystardust** [comment]
Also we can make the process menu independent ( for dmenu and rofi users)

**Comment by pystardust** [comment]
I have done some editing and merged

**Comment by Euro20179** [comment]
> @Euro20179
> This is a great idea. What if we make this in built with option `-l`?
> So we get a search loop and then if you hit escape it asks you for a search. If you hit escape again it exits.
> Even if we implement it as you say, we can make it so that loop is made 1, so that giving option `-s` would suffice with out `-l`

I think -s without -l is good (i know you have implemented it I just feel the need to reply)

**Pull request merged**